### ISSUE TITLE
Teal support?

### ISSUE THREAD

**Issue by mtdowling** [lunarmodules/ldoc#412]
Ldoc has [Moonscript support](https://lunarmodules.github.io/ldoc/manual/manual.md.html#Moonscript_Support); is there any interest in [Teal](https://github.com/teal-language/tl) support?

**Comment by alerque** [comment]
Interest? Sure. Were you considering contributing?

**Comment by catwell** [comment]
For what it's worth I have stopped using LDoc but I do use Teal, so if you contribute support I can help with reviews and testing.

**Comment by mtdowling** [comment]
Yeah, I was wondering if anyone has already put any thought into whether it would make sense to integrate Teal support with ldoc, or if Teal is different enough from the ldoc data model that it would make more sense to build a dedicated tool. Or if other similar Lua like languages have been integrated with ldoc, or if they tried and failed.
